benchmark/http_simple_auto.js

```diff
@@ -47,8 +47,8 @@ var server = http.createServer(function(req, res) {
     n = parseInt(arg, 10);
     if (n <= 0) throw new Error('bytes called with n <= 0');
     if (storedBuffer[n] === undefined) {
-      storedBuffer[n] = new Buffer(n);
-      for (i = 0; i < n; i++) {
+      storedBuffer[n] = Buffer.allocUnsafe(n);
+      for (var i = 0; i < n; i++) {
```

Comment:
Not sure why it was changed in _this_ place (it's not even the first line where it's used, but that's code style only).
Also note that no similar change was performed in `benchmark/http_simple.js`, where `i` is also not declared.

It shouldn't hold landing this PR, though, ignore this nit.